Dockerfile

```diff
@@ -16,15 +16,17 @@ WORKDIR /tmp
 ENV JENKINS_WAR_URL https://updates.jenkins-ci.org/download/war/1.642.1/jenkins.war
 ENV JENKINS_STAGING /var/jenkins_staging
 ENV JENKINS_HOME /var/jenkins_home
-ENV CATALINA_HOME /usr/local/tomcat
-ENV PATH "${CATALINA_HOME}/bin:${PATH}"
-ENV JAVA_HOME "/usr/lib/jvm/java-8-openjdk-amd64"
+ENV JENKINS_WAR /usr/share/jenkins/jenkins.war
+ENV JAVA_HOME "/usr/lib/jvm/java-7-openjdk-amd64"
 
+RUN apt-get update
+RUN apt-get install -y git python zip curl default-jre
```

Comment:
I noticed `jq` is missing from this list. We'll want to make sure that's installed since it's used by `plugin_install.sh` now.